docs/actions/view-action.md

```diff
@@ -0,0 +1,30 @@
+---
+title: View details of an Action
+intro: In this section you will learn how to find and understand details of a Devopness action
+links:
+    overview:
+    quickstart:
+    previous: /docs/actions
+    next:
+    guides:
+    related:
+        - /docs/actions
+    featured:
+---
+
+You can access details of all actions executed in an environment to which you have access.
+
+1. On Devopness, navigate to a project then select an environment
+1. On the upper-right corner, click `ACTIONS` to see a list of all actions created in the current environment
+1. In the list of actions, find the action you want to inspect and click the link in the `ACTION` column
+1. You will be redirected to the action details view
```

Comment:
This step seems like a side effect of the previous step.

Could you consider formatting the step like [Cancel Active Subcription Plan](https://www.devopness.com/docs/users/subscriptions/cancel-subscription/) in the example below?

```suggestion
    > You will be redirected to the action details view
```

### More info

https://github.com/devopness/devopness/blob/fb0f769accab88fa9128dcdb7c3a6be164818af4/docs/users/subscriptions/cancel-subscription.md?plain=1#L15-L16

BEFORE:

![image](https://github.com/user-attachments/assets/a0b3da03-681e-4420-8ee0-c2973b2f2e0f)

AFTER:

![image](https://github.com/user-attachments/assets/9052e8e0-5eb7-46d9-806a-27e626bf3bbe)